docs/drivers/aws.md

```diff
@@ -0,0 +1,74 @@
+<!--[metadata]>
++++
+title = "Amazon Web Services"
+description = "Amazon Web Services driver for machine"
+keywords = ["machine, Amazon Web Services, driver"]
+[menu.main]
+parent="smn_machine_drivers"
++++
+<![end-metadata]-->
+
+# Amazon Web Services
+Create machines on [Amazon Web Services](http://aws.amazon.com). You will need
+an Access Key ID, Secret Access Key and a VPC ID. To find the VPC ID, login to
+the AWS console and go to Services -> VPC -> Your VPCs. Select the one where you
```

Comment:
@thaJeztah seems to be:

![image](https://cloud.githubusercontent.com/assets/1347209/8683440/b7a28240-2a27-11e5-81b2-b5266f796c04.png)